src/theme.ts

```diff
@@ -289,3 +289,18 @@ export const newTheme = extendTheme({
     },
   },
 });
+
+console.log(newTheme.components);
+
+extendTheme({
+  components: {
+    MuiCard: {
+      styleOverrides: {
+        root: ({ theme }) => ({
```

Comment:
This is my initial experimentation with overriding card styling, not working currently, something to do with my use of theme, I tried newTheme and theme here, based directly on the docs I found, but it doesn't seem to be working, more investigation needed...